frontend_lib/test/component/FileDropzone.spec.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { expect } from 'chai'
+import { mount } from 'enzyme'
+import FileDropzone from '../../src/component/FileDropzone/FileDropzone.jsx'
+import { FILE_PREVIEW_STATE } from '../../src/helper.js'
+import sinon from 'sinon'
+require('../../src/component/FileDropzone/FileDropzone.styl')
+
+describe('<FileDropZone />', () => {
+  const onDropCallBack = sinon.spy()
+  const onClickCallBack = sinon.spy()
+
+  const props = {
+    onDrop: onDropCallBack,
+    onClick: onClickCallBack,
+    multipleFiles: true,
+    filename: 'randomPreview',
+    preview: FILE_PREVIEW_STATE.NO_FILE
+  }
+
+  const wrapper = mount(
```

Comment:
is mount required instead of shallow ?

Reply:
Yes because the component FileDropezone is wrapped with a react-dropzone/Dropezone so if it use shallow, we don't have have access to what Dropzone render

Reply:
ok. Add a comment explaining why you use mount()

Reply:
Btw, FileDropzone isn't wrapped by Dropzone. It has a react-dropzone/Dropzone component that wrap the jsx inside it. But FileDropzone is the hole thing.
It is more accurate to say that the the react-dropzone/Dropzone is wrapped by our FileDropzone component.